impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.css

```diff
@@ -2,26 +2,110 @@
   display: none !important;
 }
 
+.responsive.pentaho-common-dialog.filter-dialog .dijitDialogPaneContent{
+  overflow: auto !important;
+}
+
```

Comment:
This stylesheet is shared by responsive and non-responsive themes. Also, the classes assigned to HTML elements were changed. Are the non-responsive themes (crystal and sapphire) still displaying "correctly"?